v1/src/main/java/com/google/cloud/teleport/spanner/ddl/ForeignKey.java

```diff
@@ -133,6 +137,9 @@ private void prettyPrint(Appendable appendable) throws IOException {
               "Foreign Key action not supported: " + action.get().getSqlString());
       }
     }
+    if (isEnforced() != null && !isEnforced()) {
+      appendable.append(" NOT ENFORCED");
```

Comment:
I don't see dialect being used here or anywhere in this file. Is the SQL statements to create FKs same for both GSQL and postgres?

Is this feature going to be released for both GSQL and PG together?

If this feature is not supported for Postgres, can you add a check here to append " NOT ENFORCED" only for GSQL and put a TODO comment to remove the dialect check once it is supported for PG.